articles/precarious-data.md

```diff
@@ -1,51 +1,217 @@
+
+# Why the Established Tools Aren't Good Enough
+
+All this talk of decentralization and content-addressing might sound excessive. It's a major change from the way we've been doing things for the past 15 years.
+
+## What's wrong with just moving the data to a new, trusted location?
+
+Merely moving the data to a new location is not enough because it perpetuates all the problems of location-addressing. It brings all the pain and inconvenience of breaking the location-based links we've been relying on but doesn't bring any of the benefits of switching to a content-addressed approach.
+
+## Why isn't it enough to have everyone download copies of the data?
+
+Lots of Copies Keep Stuff Safe, but simply downloading copies of the data to many locations is basically adopting a decentralized approach without using any of the tools of decentralization. You need a content-addressed approach in order to answer basic questions like "Who has copies of the data?", "Are these two copies of the data identical?" and in order to communicate things like "Here is the latest version of the data" and "I have the last three versions of the data. Which one do you want?"
+
+## Can't we use the cloud to back up the data?
+
+![Sticker: "There is no Cloud. It's just someone else's computer."](https://d21ii91i3y6o6h.cloudfront.net/gallery_images/from_proof/3442/large/1418280711/die-cut-stickers.png)
+
+Many people associate cloud services with data backup. It's easy to dump copies of your data onto any number of cloud services, and most cloud providers are eager to sell the idea that their services are great for backup and disaster recovery.
+
+The good news: This means it's easy, and relatively cheap, for almost anyone to provision any amount of storage.  The bad news: this doesn't solve the fundamental problem on its own.
+
+As the designer Chris Watterston expressed so aptly in a [popular sticker](https://www.chriswatterston.com/blog/my-there-no-cloud-sticker), "There is no Cloud. It's just someone else's computer." Moving data onto cloud services does not solve the problem of centralization. It simply makes it easy for us to provision storage space and/or compute power in a range of locations. That's a powerful tool, but it's not the complete solution.
 
+## Can Libraries Save the Day?
 
```

Comment:
Might be good to add more of a concluding chapter. Also - is this for librarians? This is the second mention of librarians in the document.